pages.es/linux/torify.md

```diff
@@ -0,0 +1,32 @@
+# torify
+
+> Enruta el tráfico de red a través de la red Tor.
```

Comment:
Translation for this line "``> Note: This command has been deprecated, and is now a backwards-compatible wrapper of `torsocks`.``" should be added after this line.

Reply:
@kbdharun done